core/controller/src/main/scala/whisk/core/entitlement/RateThrottler.scala

```diff
@@ -67,7 +68,7 @@ private class RateInfo(maxPerMinute: Int) {
      * Increments operation count in the current time window by
      * one and checks if still below allowed max rate.
      */
-    def check(): Boolean = {
+    def check(maxPerMinute: Int): Boolean = {
```

Comment:
perhaps a comment here that the check always accepts the max allowed per minute since throttles may change dynamically. (maybe here or in the caller above.)